other/upgrade_2-1_mysql.sql

```diff
@@ -2865,6 +2865,26 @@ ALTER TABLE {$db_prefix}messages
 DROP INDEX id_topic;
 ---#
 
+---# Updating messages drop approved ix
+ALTER TABLE {$db_prefix}messages
+DROP INDEX approved;
+---#
+
+---# Updating messages drop id_board ix
+ALTER TABLE {$db_prefix}messages
+DROP INDEX id_board;
+---#
+
+---# Updating messages drop id_board ix alt name
+ALTER TABLE {$db_prefix}messages
+DROP INDEX idx_id_board;
+---#
+
+---# Updating messages add new id_board ix
+ALTER TABLE {$db_prefix}messages
+ADD INDEX idx_id_board (id_board, id_msg, approved);
```

Comment:
unique is missing here,
personaly i would change the index from unique to normal,
since we learned the id_msg is already pk.,

Reply:
Thank you.  Nice catch.